app/views/users/shared/_links.html.erb

```diff
@@ -1,17 +1,32 @@
 <div class="field form-group users-shared">
   <div class="input-group">
     <div class="row center-row users-shared-row">
-      <h6 class="users-shared-row">Log In With</h6>
+      <h6 class="users-shared-row">or</h6>
     </div>
-    <div class="row center-row">
+    <div class="row center-row auth-btns-container">
       <a href="/users/auth/google_oauth2">
-        <%= image_tag "SVGs/google-plus.svg", alt: "Google Icon", :class => "users-social-links" %>
+        <div class="google-btn auth-btn">
+          <div class="google-icon-wrapper auth-icon-wrapper">
+            <img class="google-icon auth-icon" alt="Google Icon" src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg"/>
```

Comment:
Add all required images in assets/images. And use erb syntax to use them here (Check we might already have some of these images there)